app/presenters/check_answers/work_items_card.rb

```diff
@@ -46,17 +47,52 @@ def work_item_rows
     end
 
     def total_rows
-      [
-        {
-          head_key: 'total',
-          text: work_item_total,
-          footer: true
-        }
-      ]
+      if @claim.firm_office.vat_registered == YesNoAnswer::YES.to_s
+        [
+          {
+            head_key: 'total',
+            text: work_item_total,
+            footer: true
+          },
+          {
+            head_key: 'total_inc_vat',
+            text: work_item_total_inc_vat
+          },
+        ]
+      else
+        [
+          {
+            head_key: 'total',
+            text: work_item_total,
+            footer: true
+          }
+        ]
+      end
     end
 
     def work_item_total
-      NumberTo.pounds(work_items.total_cost)
+      format_total(work_items.total_cost)
+    end
+
+    def work_item_total_inc_vat
+      format_total(work_items.work_item_forms.sum { |item| item_plus_vat(item) })
+    end
+
+    def format_total(value)
+      text = "<strong>#{currency_value(value)}</strong>"
+      ApplicationController.helpers.sanitize(text, tags: %w[strong])
+    end
+
+    def currency_value(value)
+      NumberTo.pounds(value || 0)
+    end
+
+    def item_plus_vat(item)
+      (item.total_cost * @vat_rate) + item.total_cost
```

Comment:
round need to round this here to avoid sums not equalling.. so
```
value = (item.total_cost * @vat_rate) + item.total_cost
value.round(2)
```

need this in all places...